lib/Swagger/README.md

```diff
@@ -1,42 +1,126 @@
 # Swagger
-A collection of clients for microservices annotated with [Swagger](http://swagger.io/), auto-generated using [swagger-codegen](https://github.com/swagger-api/swagger-codegen). *DO NOT* manually edit PHP classes here.
+No descripton provided (generated by Swagger Codegen https://github.com/swagger-api/swagger-codegen)
```

Comment:
changes in this file also could be reverted, for now we loose the instructions about how to generate swagger code